lib/dry/schema/message_compiler.rb

```diff
@@ -199,6 +199,14 @@ def message_tokens(args)
             hash[arg[0]] = arg[1]
           end
         end
+        append_mapped_size_tokens(tokens)
```

Comment:
@tadeusz-niemiec could you make it so that this is used only when args include `size` key? Remapping tokens even when it's not needed will have a performance impact

Reply:
@solnic done